api/src/main/java/io/strimzi/api/kafka/model/KafkaClusterRebalanceSpec.java

```diff
@@ -23,20 +23,21 @@
         builderPackage = "io.fabric8.kubernetes.api.builder"
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({ "goals", "verbose" })
+@JsonPropertyOrder({ "goals", "skipHardGoalCheck", "verbose" })
```

Comment:
So we don't currently use the verbose output option in the rebalance request. It is feasible a user may want to see the individual partition reassignments for an optimisation proposal. However, the output can be huge and may overwhelm the status field length limit. I am not saying we need to fix this now but we may need to think of a way to get this information to a user in a K8s friendly way. ConfigMap?

Reply:
Let's remember to open an issue after RC and after doing some tests to see how the status looks like when verbose is enabled.